Assets/Scripts/EOSTitleStorageManager.cs

```diff
@@ -199,204 +164,35 @@ public void ReadFile(string fileName, OnReadFileCallback ReadFileCompleted)
                     newTransfer.Data = new byte[(uint)fileMetadata?.FileSizeBytes];
 
 
-                    TransfersInProgress.Add(fileName, newTransfer);
+                    _transfersInProgress.Add(fileName, newTransfer);
```

Comment:
This can throw an error if `fileName` already exists as a key. Make sure to use the `[]` operator to avoid this.

Reply:
Fixed.